week-3/24-dietary/index.html

```diff
@@ -19,7 +19,29 @@ <h1>CodeYourFuture Summit</h1>
         </p>
 
         <form>
-          <!-- Your multiple choice goes here -->
+          <fieldset>
+            <legend>dietry requirments</legend>
+            <label>
+              <input type="checkbox" name="yumy food" value="ha" />
```

Comment:
Watch out for typos. You have two fields here, one called `yumy food` and one called `yummy food`.

Reply:
Tanks for your feedback and recommendation.i will finish the rest of exercises.